zio-http/src/main/scala/zio/http/ServerConfigLayer.scala

```diff
@@ -0,0 +1,15 @@
+package zio.http
+
+import zio.ZLayer
+
+object ServerConfigLayer {
```

Comment:
I would move all these into the `ServerConfig` companion object.

Reply:
Done